### PR TITLE
Build home and about page

### PR DESCRIPTION
**Pull request by edenlisk**
In this branch I did the following:
1. Build home page and its styling
2. Build About page and its styling
Here is the [link to video](https://youtu.be/cU7KW8GDBqQ)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  );
+  if (i < 2) {
+    performersSection.appendChild(performersContainer);
+  } else {
+    performersSection.appendChild(performersContainer);
+    performersContainer.classList.add('d-none');
+  }
+}
+
+const showmore = document.createElement('div');
```
Comment (christianbiring1) [inline index.js]: - [x] Kindly make sure on the Desktop version of the page we don't have the see-more feature. only the mobile version has this feature as it is in the provided template.

Your design :point_down: 
image

Template design :point_down: 
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        <div class="partners-logo">
+          <img class="mx-auto" src="assets/mobile/partners/microsoft.png" alt="microsoft logo" height="100" width="200">
+          <img class="mx-auto" src="./assets/mobile/partners/google.png" alt="google logo" width="200" height="85">
+          <img class="mx-auto" src="./assets/mobile/partners/pixar.png" alt="pixar logo" width="300" height="100">
+          <img class="mx-auto" src="./assets/mobile/partners/microverse.png" alt="microverse logo" width="400" height="99">
+        </div>
+      </div>
+    </section>
+  </footer>
+
```
Comment (christianbiring1) [inline index.html]: - [x] As it is in the provided template, Kindly make sure you have the footer section visible on your home page.

Your design :point_down: 
image

behance design :point_down: 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  <div>
+    <nav>
+      <header class="container-fluid header pt-0 pb-0">
+        <ul class="list-unstyled list-inline text-end header-bar lead fs-6">
+          <li class="list-inline-item"><a href="#" aria-label="Facebook"><i class="header-item bi-facebook"></i></a></li>
+          <li class="list-inline-item"><a href="#" aria-label="Twitter"><i class="header-item bi-twitter"></i></a></li>
+          <li class="list-inline-item"><a class="header-item text-decoration-none" href="#">English</a></li>
+          <li class="list-inline-item"><a class="header-item text-decoration-none" href="#">My Page</a></li>
+          <li class="list-inline-item"><a class="header-item text-decoration-none" href="#">Logout</a></li>
+        </ul>
```
Comment (christianbiring1) [inline index.html:26]: - [x] Kindly make sure you stick to the design as much as possible. `the typos, spacing(margin, padding)` of the elements on your page have to match the ones provided on the template
Your design :point_down: 
image

Template provided :point_down: 
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+            <h2 class="display-3 fw-semibold">2022</h2>
+            <p class="lead fw-semibold">Concert Festival 2022 in Moscow</p>
+          </div>
+          <div class="text-light cairo-event">
+            <h2 class="display-3 fw-semibold">2021</h2>
+            <p class="lead fw-semibold">Concert Festival 2022 in Cairo</p>
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (christianbiring1) [inline about.html]: - [x] Kindly, as is the case on the provided template, You may consider giving your image the orange background 
- [x] Also you may consider adding the footer on this page as well as is the case in the provided template.

Your design :point_down: 
image

The template :point_down: 
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+      </ul>
+    </header>
+    <nav class="bg-light d-md-flex desktop-nav">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="index.html"><img class="pt-1 ms-5" src="./assets/mobile/cflogo.png" width="40" height="40" alt="Concert Festival Logo"></a>
+      </div>
+      <div class="text-center desktop-navigation">
+        <ul class="list-unstyled fw-bold d-md-flex align-items-center gap-4 justify-content-center px-5">
+          <li><a class="text-decoration-none list-inline-item desktop-nav-item" href="about.html">About</a></li>
+          <li><a class="text-decoration-none list-inline-item desktop-nav-item" href="index.html#program">Program</a></li>
```
Comment (christianbiring1) [inline about.html]: - [x] For a better user experience, Kindly make sure when the user is navigating on the about page, they can be able to see the `home` link in the navigation bar.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+          <img class="mx-auto" src="./assets/mobile/partners/pixar.png" alt="pixar logo" width="300" height="100">
+          <img class="mx-auto" src="./assets/mobile/partners/microverse.png" alt="microverse logo" width="400" height="99">
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer class="container-fluid">
+    <div class="container text-center lead fw-semibold">
+      <p>&commat; 2023 Concert Festival All Rights Reserved</p>
+    </div>
```
Comment (christianbiring1) [inline about.html]: - [x] In the footer section we are supposed to see a logo and some text. Kindly consider fixing that.

Your design :point_down: 
image

The template design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @edenlisk , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
# STATUS : REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @edenlisk    👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**